portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,60 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+#landing-page{
```

Comment:
It's generally best practice to avoid using IDs as selectors within CSS (i.e. adding and styling the element using a class selector, i.e. `.landing-page` instead of `#landing-page`). The idea is to keep the styles reusable. There are some cases where you'll need to style on ids, but these should be rare.

A good read on ids vs classes: https://css-tricks.com/the-difference-between-id-and-class/